content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,33 +1,41 @@
+While a deployment framework is not required to deploy to AWS Lambda, this guide covers deploying with [AWS Serverless Application Model (SAM)](https://aws.amazon.com/serverless/sam/), [Serverless Framework](https://www.serverless.com/framework/), and [SST](https://sst.dev/):
+
+- AWS SAM is a open-source framework from AWS that can be used in the creation of serverless applications. AWS SAM includes the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-reference.html#serverless-sam-cli) which you can use to build, test, and deploy your application.
+- Serverless Framework provides a CLI that helps with workflow automation and AWS resource provisioning. While Prisma works well with the Serverless Framework "out of the box", there are a few improvements that can be made within your project to ensure a smooth deployment and performance. There is also additional configuration that is needed if you are using the [`serverless-webpack`](https://www.npmjs.com/package/serverless-webpack) or [`serverless-bundle`](https://www.npmjs.com/package/serverless-bundle) libraries.
+- SST provides tools that make it easy for developers to define, test, debug, and deploy their application. Prisma works well with SST, but must be configured so that your schema is correctly packaged by SST.
 
 </TopBlock>
 
-## Prerequisites
+## General considerations when deploying to AWS Lambda
+
+This section covers changes you will need to make to your application, regardless of framework. After following these steps, follow the steps for your framework.
 
-This guide assumes you have an existing project using both Serverless Framework and Prisma. For a guide on how to get started with Serverless Framework, we recommend [Setting Up Serverless Framework With AWS](https://www.serverless.com/framework/docs/getting-started).
+- [Deploying with AWS SAM](#deploying-with-aws-sam)
+- [Deploying with the Serverless Framework](#deploying-with-the-serverless-framework)
+- [Deploying with SST](#deploying-with-sst)
 
-## Binary targets in <inlinecode>schema.prisma</inlinecode>
+### Define binary targets in Prisma Schema
 
 The Prisma schema should contain the following in the `generator` block:
 
 ```prisma
 binaryTargets = ["native", "rhel-openssl-1.0.x"]
 ```
 
-This is necessary because the runtime used in the development and when your application is deployed differ. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine file available.
+This is necessary because the runtimes used during development and after deployment differ. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine file available.
```

Comment:
```suggestion
This is necessary because the runtimes used in development and deployment differ. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine file available.
```